hash/spectrum_cass.xml

```diff
@@ -76610,27 +76700,38 @@ license:CC0
 	</software>
 
 	<software name="gotojail">
-		<description>Go To Jail - Release 1</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<description>Go To Jail (release 1)</description>
+		<year>1983</year>
+		<publisher>Automata UK</publisher>
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="41538">
-				<rom name="go to jail - release 1.tzx" size="41538" crc="da83cd9d" sha1="791b42fc0b7d85665490dd645018331a297b5567"/>
+				<rom name="go to jail - release 1.tzx" size="41538" crc="655ba6a6" sha1="933f7e9e7288f5062fef598528292f5f82d12a75"/>
 			</dataarea>
 		</part>
 	</software>
 
 	<software name="gotojail_a" cloneof="gotojail">
-		<description>Go To Jail - Release 2</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
+		<description>Go To Jail (release 2)</description>
+		<year>1983</year>
+		<publisher>Automata UK</publisher>
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="40700">
 				<rom name="go to jail - release 2.tzx" size="40700" crc="fef9d31f" sha1="6f17ed41128fb2be2e338c22d7aa3d6e42307824"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<software name="gotojail_b" cloneof="gotojail">
+		<description>Gehen Sie in das Defaengnis</description>
```

Comment:
There's a typo here. Also, the original cassette inlay does use the "ä" character:

https://ia800604.us.archive.org/view_archive.php?archive=/1/items/World_of_Spectrum_June_2017_Mirror/World%20of%20Spectrum%20June%202017%20Mirror.zip&file=World%20of%20Spectrum%20June%202017%20Mirror/sinclair/games-inlays/Rereleases/g/GoToJail(GehenSieInDasGefangnis)(Wicosoft).jpg

Reply:
Typo corrected. Thanks!